tests/test_config_management.py

```
@@ -903,16 +903,16 @@ def test_parallel_processor_sequential_and_thread_modes(tmp_path: Path) -> None:
+        return None, None, None
 
     p_seq = ParallelProcessor(max_workers=1, use_processes=False, enabled=True)
-    _, errors = p_seq.process_files(files, process_func)
+    _, errors, skips = p_seq.process_files(files, process_func)
     assert p_seq.enabled is False
     assert len(errors) == 1
     assert isinstance(errors[0], FileAnalysisError)
 
     p_thr = ParallelProcessor(max_workers=2, use_processes=False, enabled=True)
-    results, errors = p_thr.process_files(files, lambda p: (None, None))
+    results, errors, skips = p_thr.process_files(files, lambda p: (None, None, None))
```

Ruff (0.15.9)

[warning] Unpacked variable `skips` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)